custom/icds_reports/static/js/directives/month-filter/month-filter.directive.js

```diff
@@ -28,7 +28,8 @@ function MonthModalController($location, $uibModalInstance) {
     vm.selectedMonth = $location.search()['month'] !== void(0) ? parseInt($location.search()['month']) : new Date().getMonth() + 1;
     vm.selectedYear = $location.search()['year'] !== void(0) ? parseInt($location.search()['year']) : new Date().getFullYear();
 
-    if (isSDD && (vm.selectedYear < 2019 || (vm.selectedYear === 2019 && vm.selectedMonth === 1))) {
+    var isSSDNotAvailableForGivenDate = isSDD && (vm.selectedYear < 2019 || (vm.selectedYear === 2019 && vm.selectedMonth === 1));
+    if (isSSDNotAvailableForGivenDate) {
         vm.showMessage = true;
```

Comment:
nit: I think this could likely be renamed to something clearer (like `invalidDateSet`) since it is now controlling more than just the message (and anyways "message" is very vague)